doc-content/shared-kie-docs/src/main/asciidoc/Workbench/RemoteAPI/security-management-rest-api-supported-permissions-ref.adoc

```diff
@@ -88,12 +88,14 @@ The `name` attribute is an identifier of a resource that you add as an exception
 "pages": {
 	"create": true,
 	"read": false,
-	"exceptions": [{
+	"exceptions": [
+    {
```

Comment:
This should be a little bit to the right.

Reply:
Done